public/components/integrations/components/available_integration_overview_page.tsx

```diff
@@ -88,6 +85,45 @@ export function AvailableIntegrationOverviewPage(props: AvailableIntegrationOver
     </EuiFilterButton>
   );
 
+  return (
+    <EuiFilterGroup>
+      <EuiPopover
+        id="popoverExampleMultiSelect"
+        button={button}
+        isOpen={isPopoverOpen}
+        closePopover={closePopover}
+        panelPaddingSize="none"
+      >
+        <EuiPopoverTitle paddingSize="s">
+          <EuiFieldSearch compressed />
+        </EuiPopoverTitle>
+        <div className="ouiFilterSelect__items">
+          {items.map((item, index) => (
+            <EuiFilterSelectItem
+              checked={item.checked ? 'on' : undefined}
+              key={index}
+              onClick={() => updateItem(index)}
+            >
+              {item.name}
+            </EuiFilterSelectItem>
+          ))}
+        </div>
+      </EuiPopover>
+    </EuiFilterGroup>
+  );
+};
+
+export function AvailableIntegrationOverviewPage(props: AvailableIntegrationOverviewPageProps) {
```

Comment:
To support flyout installation, I'm refactoring the existing overview component to support more optional functionality. Then using the flyout install can be done by:
- Disabling custom category filtering (supply constant filters based on data source type)
- Disable card/table toggle
- (Future PR) support loading install form directly instead of linking to its dedicated page